tests/system/test_sqlalchemy_bigquery.py

```diff
@@ -691,3 +691,37 @@ def test_has_table(engine, engine_using_test_dataset, bigquery_dataset):
     assert engine_using_test_dataset.has_table(f"{bigquery_dataset}.sample") is True
 
     assert engine_using_test_dataset.has_table("sample_alt") is False
+
+
+def test_distinct_188(engine, bigquery_dataset, metadata):
```

Comment:
The `metadata` fixture doesn't look to be used here.

Reply:
Right you are! I'll remove it.

Reply:
done